docs/penguins.nim

```diff
@@ -52,34 +54,34 @@ nbImage(url="images/penguins_map.png",
+#  ggplot(df, aes(x="body_mass_g", y="flipper_length_mm", color = "sex", shape="species")) + geom_point() + ggsave("images/penguins_mass_vs_length_with_sex_and_species2.png")
+#nbImage(url="images/penguins_mass_vs_length_with_sex_and_species2.png", caption="Penguins' mass vs flipper length")
 nbCode:
-  let df1 = df.filter(f{c"body_mass_g" != "NA"}) # why c""?
-  ggplot(df1, aes(x="body_mass_g", y="flipper_length_mm", color = "sex")) + geom_point() + ggsave("images/penguins_mass_vs_length_with_sex.png")
+  let df1 = df.filter(f{`body_mass_g` != "NA"}) # c"foo" == `foo` == idx("foo") (accent quoted not usable for columns w/ spaces)
+  ggplot(df1, aes(x="body_mass_g", y="flipper_length_mm", color = "sex", shape="species")) + geom_point() + ggsave("images/penguins_mass_vs_length_with_sex.png")
 nbImage(url="images/penguins_mass_vs_length_with_sex.png", caption="Penguins' mass vs flipper length (colored by sex)")
 nbText: """A few things to remark:
 
 - as expected body mass and flipper length are linearly correlated
 - males are in general bigger than females but there appear 2 groups, possibly related to species
-- we have some more NAs (and one '.') in sex column (even after filtering for NAs in numeric columns)
-
-Let's now look at the same chart coloring by species instead of by sex (shape is not yet supported in ggplotnim):
-"""
-nbCode:
-  ggplot(df1, aes(x="body_mass_g", y="flipper_length_mm", color = "species")) + geom_point() + ggsave("images/penguins_mass_vs_length_with_species.png")
-nbImage(url="images/penguins_mass_vs_length_with_species.png", caption="Penguins' mass vs flipper length (colored by species)")
-nbText: """We can see that sizes of Adelie and Chinstrap overlap, while Gentoo penguins are in general bigger.
+- we have some more NAs (and one '.') in sex column (even after filtering for NAs in numeric columns
+- we can see that sizes of Adelie and Chinstrap overlap, while Gentoo penguins are in general bigger
 
-As a final plot, I would like to (partly) reproduce a plot that "shows" the presence of [Simpson's paradox](https://en.wikipedia.org/wiki/Simpson%27s_paradox) in this dataset,
+As a final plot, I would like to (partly, confidence bands are missing) reproduce a plot that "shows" the presence of [Simpson's paradox](https://en.wikipedia.org/wiki/Simpson%27s_paradox) in this dataset,
 as reported by [this tweet](https://twitter.com/andrewheiss/status/1301166792627421186):
 """
 nbCode:
-  ggplot(df1, aes(x="culmen_depth_mm", y="body_mass_g", color = "species")) + geom_point() + ggsave("images/penguins_simpson.png")
+  ggplot(df1, aes(x="culmen_depth_mm", y="body_mass_g")) +
+    geom_point(aes = aes(color = "species")) +      # point...
+    geom_smooth(aes = aes(color = "species"),       # and smooth classified by species
+                smoother = "poly", polyOrder = 1) + # polynomial order 1 == line
+    geom_smooth(smoother = "poly", polyOrder = 1) + # and smooth w/o classification by species
```

Comment:
```suggestion
    geom_smooth(smoother = "poly", polyOrder = 1) + # and smooth without classification by species
```
as above, I always prefer the full with/without expression, better for those not familiar with the abbreviation